src/HashBus.Twitter.BackFill/App.cs

```diff
@@ -29,7 +33,14 @@ class App
                     TweetId = tweetId,
                 };
 
-                bus.Send(command);
+                await endpointInstance.Send(command).ConfigureAwait(false);
+
+                await Task.Delay(Timeout.Infinite);

```

Comment:
This line should not be here. The app should send the message and exit, just like in the sync version.